src/localization/buttonStrings.js

```diff
@@ -27,9 +27,9 @@ export const buttonStrings = new LocalizedStrings({
     view_regimen_data: 'View Regimen Data',
   },
   fr: {
-    // TODO: view_regimen_data
+    view_regimen_data: 'Données de régimes',
     add_batch: 'Ajouter un lot',
-    add_master_list_items: "Ajouter une liste-Maître d'articles ?",
+    add_master_list_items: 'Ajouter à\npartir des\nlistes maîtres',
```

Comment:
I think can get away with a single newline here:

```suggestion
    add_master_list_items: 'Ajouter à partir\ndes listes maîtres',
```

Before:

![#1400-column-header-spacing-1](https://user-images.githubusercontent.com/43223637/67170347-bb8f6580-f40d-11e9-9183-e70524307e79.png)

After:

![#1400-column-header-spacing-2](https://user-images.githubusercontent.com/43223637/67170353-c0541980-f40d-11e9-8a69-37b466fa15dc.png)

Reply:
Ah nice thanks - f83afcc